### PR TITLE
#315 - Firewall-applet allows to use KDE network connection editor

### PR DESCRIPTION
**Pull request by Zesko**
Fixed: https://github.com/firewalld/firewalld/issues/315

Firewall-applet allows to open KDE network connection editor.

### CODE REVIEW

File: src/firewall-applet.in [hunk 1]
```diff
@@ -58,14 +58,19 @@ def search_app(app):
...
+                "/usr/bin/nm-connection-editor",
                     "/bin/nm-connection-editor",
                 "/usr/bin/kde5-nm-connection-editor",
                     "/bin/kde5-nm-connection-editor",
                 "/usr/bin/kde-nm-connection-editor",
                     "/bin/kde-nm-connection-editor" ]:
     if os.path.exists(binary):
-        NM_CONNECTION_EDITOR = binary
+        if binary.endswith("systemsettings"):
+            NM_CONNECTION_EDITOR = binary + " kcm_networkmanagement"
```
Comment (erig0) [inline src/firewall-applet.in:71]: Why append "kcm_networkmanagement" here instead of in the array entry?

Comment (Zesko) [reply src/firewall-applet.in]: Changed, done!

File: src/firewall-applet.in [hunk 2]
```diff
@@ -920,12 +922,17 @@ class TrayApplet(QtWidgets.QSystemTrayIcon):
             return
 
         if uuid:
-            if "kde-" in NM_CONNECTION_EDITOR:
+            if "systemsettings" in NM_CONNECTION_EDITOR:
+                os.system("%s kcm_networkmanagement --args Uuid=%s" % (NM_CONNECTION_EDITOR, uuid))
```
Comment (erig0) [inline src/firewall-applet.in:926]: Can you confirm this does NOT need the ampersand `&` ?

Comment (Zesko) [reply src/firewall-applet.in:926]: Oh, It needs the ampersand `&` to create a new process. The main process doesn't have to wait for this process.

Comment (Zesko) [reply src/firewall-applet.in:926]: Fixed

Comment (Zesko) [reply src/firewall-applet.in:926]: I can't do git push, you can fix it as you know.

Comment (erig0) [reply src/firewall-applet.in:926]: I'll add it and push. I'm also squashing you commits and fixing the commit message.

**Review by erig0 — APPROVED** [review]
lgtm, will merge after CI completes.

**Pull request merged**